test/api/test_config.py

```diff
@@ -0,0 +1,168 @@
+"""
+Copyright (c) 2020 Genome Research Limited
+
+Author: Aiden Neale <an12@sanger.ac.uk>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or (at
+your option) any later version.
+
+This program is distributed in the hope that it will be useful, but
+WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see https://www.gnu.org/licenses/
+"""
+
+import os
+import unittest
+from tempfile import TemporaryDirectory
+from core import typing as T, time
```

Comment:
It's good style to separate different "classes" of imports, ordering each class alphabetically (with first-class import statements coming first). First would come standard library imports, then external library imports, followed finally by internal module imports. You have the classes and sort order right here, but a newline between the classes (standard library and internal modules, in this case) is a good visual indicator.